client/src/cli/deps.ts

```diff
@@ -0,0 +1,2 @@
+export { err, map, ok, Result, toString } from "npm:true-myth@6.2.0/result";
```

Comment:
it's a bit strange we use `deps.ts` only sometimes
no strong opinion about this just thinking aloud
not sure what is better, fine with leaving as is